core/src/layers/prometheus.rs

```diff
@@ -88,12 +88,62 @@ use crate::*;
 #[derive(Default, Debug, Clone)]
 pub struct PrometheusLayer {
     registry: Registry,
+    requests_duration_seconds_buckets: Vec<f64>,
+    bytes_total_buckets: Vec<f64>,
 }
 
 impl PrometheusLayer {
     /// create PrometheusLayer by incoming registry.
+    /// For promtheus layer, we will use default buckets for HistogramMetric.
```

Comment:
We can place those comments for `PrometheusLayer` as an example maybe. And it's better to document the default value for `requests_duration_seconds_buckets` and `bytes_total_buckets` there too.